specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -562,7 +555,6 @@
           "MachineLearningComputes"
         ],
         "operationId": "MachineLearningCompute_Delete",
-        "x-ms-long-running-operation": true,
```

Comment:
Why is this not long running? 201 status corresponds to `Created` which is incorrect for a `DELETE` operation

Reply:
they were not on our internal swagger. I can add the long-running back.

Delete is returning a 202

Reply:
added long-running back. Update accordingly